pages/get-started.mdx

```diff
@@ -7,6 +7,98 @@ export const meta = {
 
 ## Get started with Vert.x
 
-TODO
+In this guide you'll learn how to get started with a new Vert.x project.
 
-test (Julien)
+Before starting, you need:
+
+* A JDK 1.8 or higher
+* A text editor or IDE
+* Maven
```

Comment:
Do you need to have a version for Maven?

Reply:
I suppose Maven 3+ should be fine, let me add it